ariadne/executable_schema.py

```diff
@@ -40,6 +43,9 @@ def make_executable_schema(
     validate_schema_enum_values(schema)
     repair_default_enum_values(schema, flat_bindables)
 
+    if kwargs_to_snake_case:
```

Comment:
I'm curious if this should be an option or just a utility that you can run against your schema separately. But I guess the latter would require us to create a utility that copies whole schema so we don't mutate it.